src/cryptoadvance/specter/cli/cli_server.py

```diff
@@ -136,7 +136,13 @@ def server(
                 if os.path.isfile(filename):
                     extra_files.append(filename)
 
-    kwargs = {"host": host, "port": app.config["PORT"], "extra_files": extra_files}
+    # 'use_reloader': False prevents that a..run runs twice when in debug mode. See https://stackoverflow.com/questions/9449101/how-to-stop-flask-from-initialising-twice-in-debug-mode
```

Comment:
But disabling the reloader disables the ability on hot reloading, right? What's the problem of initialising twice?
Can't see the positive trade-off here (and disabling the reloader without being able to switch it on again is imho not a good idea).

Reply:
I was (and am) unclear why in debug the app is initialized twice.   What is hot-reloading?

The second initialization then tried to open a second websocket server on the same port. That of course lead to an error (in debug only).

Reply:
If you chnage something in `templates` the change take effect immediately. That's not true for e.g. `notification_manager.py`. Here it will only work if the reloader is switched on which is obviously very usefull in dev-mode.
That it's loaded twice is a very old issue which i don't understand either.

Maybe we can detect whether we're in the right thread and suppress it in the wrong one?
https://stackoverflow.com/questions/25504149/why-does-running-the-flask-dev-server-run-itself-twice

Reply:
- [ ] remove the `use_reloader=False` and prevent the webserver/client from starting twice. Maybe via a global lock?